website/docs/signing.md

```diff
@@ -1,5 +1,7 @@
 # Signing Packages
 
+***Note:*** Avalible on dalec release `v0.3.0+`
```

Comment:
```suggestion
***Note:*** Available with Dalec release `v0.3.0+`
```

Reply:
I'll open an issue for supporting versioned docs so we don't have to do this in the future